modules/openapi-generator/src/main/resources/dart2/class.mustache

```diff
@@ -109,15 +109,15 @@ class {{{classname}}} {
           ? null
           : (json[r'{{{baseName}}}'] as List).map(
       {{#items.complexType}}
-              {{items.complexType}}.listFromJson(json[r'{{{baseName}}}'])
+              {{items.complexType}}.listFromJson(json[r'{{{baseName}}}'] as List<dynamic>)
```

Comment:
same here, we know that the method accepts `List<dynamic> json` as first param, so why the cast here?

Reply:
same as above, with the `implicit-casts:false` we cant implicit cast `dynamic` to `List<dynamic>`, but as I said in the description, if that flag is not desirable I can remove almost all casts :thinking: and I'm pretty sure it will still work